getting_started.md

```diff
@@ -21,7 +21,7 @@ source activate [YOUR_CDAT_CONDA_ENV]
 
 Once you've loaded the environment, you should be able to run the examples. They should output a .png file that has the same image as the example.
 
-We strongly recommend using Jupyter notebook for the tutotrials
+We strongly recommend using Jupyter notebook for the tutotrials.  When you type the command below it is best to have navigated to a folder that contains at least one jupyter notebook (file extention .ipynb).
```

Comment:
typo on `tutorials` and actually in notebook you can't go up so it would say something like:
When you type the command below it is best to have navigated to a folder containing the jupyter notebooks sub-folders.

Reply:
I've fixed the typo and good point on not being able to navigate to a higher folder after launching the Jupyter Notebook app. See if the way I've rephrased things makes sense now. If not, I can edit some more.